From: Anonymous <noreply@weblate.org>
Date: Fri, 28 Aug 2026 07:21:23 +0000
Subject: =?utf-8?b?VHJhbnNsYXRlZCB1c2luZyBXZWJsYXRlICjmnJ3pspzor60gKGtvX0tQ?=
 =?utf-8?b?KSk=?=

Currently translated at 100.0% (15 of 15 strings)

Translation: openkylin-huanghe/kylin-os-manager-tool-box
Translate-URL: http://weblate.openkylin.top/projects/openkylin-huanghe/kylin-os-manager-tool-box/ko_KP/
---
 plugins/tool_box/translations/tool-box_ko_KP.ts | 30 ++++++++++++-------------
 1 file changed, 15 insertions(+), 15 deletions(-)

diff --git a/plugins/tool_box/translations/tool-box_ko_KP.ts b/plugins/tool_box/translations/tool-box_ko_KP.ts
index f6c385a..17be41b 100644
--- a/plugins/tool_box/translations/tool-box_ko_KP.ts
+++ b/plugins/tool_box/translations/tool-box_ko_KP.ts
@@ -6,7 +6,7 @@
     <message>
         <location filename="../tool_box.cpp" line="32"/>
         <source>ToolBox</source>
-        <translation type="unfinished"></translation>
+        <translation>도구 상자</translation>
     </message>
 </context>
 <context>
@@ -15,49 +15,49 @@
         <location filename="../app_item.cpp" line="340"/>
         <location filename="../app_item.cpp" line="597"/>
         <source>Getting installation package status, please try again later</source>
-        <translation type="unfinished"></translation>
+        <translation>설치 패키지 상태를 불러오는 중입니다. 잠시 후 다시 시도해 주십시오.</translation>
     </message>
     <message>
         <location filename="../app_item.cpp" line="343"/>
         <location filename="../app_item.cpp" line="387"/>
         <location filename="../app_item.cpp" line="600"/>
         <source>Please proceed with installation first</source>
-        <translation type="unfinished"></translation>
+        <translation>먼저 설치를 진행해 주십시오</translation>
     </message>
     <message>
         <location filename="../app_item.cpp" line="363"/>
         <source>No application detected</source>
-        <translation type="unfinished"></translation>
+        <translation>해당 응용 프로그람을 검출하지 못했습니다.</translation>
     </message>
     <message>
         <location filename="../app_item.cpp" line="511"/>
         <source>Add to desktop shortcut</source>
-        <translation type="unfinished"></translation>
+        <translation>바탕화면 바로가기에 추가하기</translation>
     </message>
     <message>
         <location filename="../app_item.cpp" line="546"/>
         <source>Kylin Os Manager</source>
-        <translation type="unfinished"></translation>
+        <translation>키린 매니저</translation>
     </message>
     <message>
         <location filename="../app_item.cpp" line="550"/>
         <source>Installed successfully</source>
-        <translation type="unfinished"></translation>
+        <translation>설치 성공</translation>
     </message>
     <message>
         <location filename="../app_item.cpp" line="551"/>
         <source>The %1 has been successfully installed.</source>
-        <translation type="unfinished"></translation>
+        <translation>%1 설치가 성공하였습니다.</translation>
     </message>
     <message>
         <location filename="../app_item.cpp" line="556"/>
         <source>Installation failed</source>
-        <translation type="unfinished"></translation>
+        <translation>설치 실패</translation>
     </message>
     <message>
         <location filename="../app_item.cpp" line="557"/>
         <source>The %1 was not successfully installed. Please try again later.</source>
-        <translation type="unfinished"></translation>
+        <translation>%1 설치가 실패하였습니다. 잠시 후 다시 시도해 주십시오.</translation>
     </message>
 </context>
 <context>
@@ -65,7 +65,7 @@
     <message>
         <location filename="../blankpage.cpp" line="22"/>
         <source>No tools found</source>
-        <translation type="unfinished"></translation>
+        <translation>내용 없음</translation>
     </message>
 </context>
 <context>
@@ -74,25 +74,25 @@
         <location filename="../mainwindow.cpp" line="55"/>
         <location filename="../mainwindow.cpp" line="142"/>
         <source>Feature tools</source>
-        <translation type="unfinished"></translation>
+        <translation>기능 도구</translation>
     </message>
     <message>
         <location filename="../mainwindow.cpp" line="64"/>
         <location filename="../mainwindow.cpp" line="147"/>
         <source>Debug tools</source>
-        <translation type="unfinished"></translation>
+        <translation>운영 디버그 도구</translation>
     </message>
     <message>
         <location filename="../mainwindow.cpp" line="73"/>
         <location filename="../mainwindow.cpp" line="152"/>
         <source>Troubleshooting tools</source>
-        <translation type="unfinished"></translation>
+        <translation>고장 수리</translation>
     </message>
     <message>
         <location filename="../mainwindow.cpp" line="82"/>
         <location filename="../mainwindow.cpp" line="157"/>
         <source>Other tools</source>
-        <translation type="unfinished"></translation>
+        <translation>기타</translation>
     </message>
 </context>
 </TS>
